test/integration/patients/sidebar/action-sidebar.js

```diff
@@ -202,7 +202,7 @@ context('action sidebar', function() {
       .should('not.exist');
   });
 
-  specify('display action sidebar', function() {
+  specify.only('display action sidebar', function() {
```

Comment:
remove .only

Reply:
It is one way to make the tests run faster 🙃 (fixed in latest push)

Reply:
"QA engineers hate him. Find out this dev's brilliant secret to speeding up integration tests!"